api/groups/transactionGroup.go

```diff
@@ -30,6 +30,7 @@ const (
 	sendMultiplePath                 = "/send-multiple"
 	getTransactionPath               = "/:txhash"
 	getTransactionsPool              = "/pool"
+	getTransactionsForSender         = "/for-sender/:sender"
```

Comment:
it is not clear that these are the transactions from pool

Reply:
renamed to `/pool/:sender`